frontend/components/Index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from "react-router-dom";
+import Main from 'components/Main';
+
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Suggest to rename it to app.jsx 
and take out the part with inserting react in root to packs/application.jsx, so here will be the only component